src/legacy/Legacy.routes.tsx

```diff
@@ -1,52 +1,45 @@
-import * as React from 'react';
-import { Route, Switch } from 'react-router-dom';
+// import * as React from 'react';
```

Comment:
This file looks all commented out; should it be deleted?

Reply:
I was thinking about deleting it in another pr. Wanted to make sure that I wasn't deleting anything that might need to be referenced later but we can always check the git tag.  Will delete here 👍